src/components/Hero/Hero.style.js

```diff
@@ -1,25 +1,50 @@
+import { string, func, oneOfType, oneOf, any } from 'prop-types';
 import styled from 'styled-components';
 import { rem } from 'polished';
-import { variables } from '../../global/helpers';
+import { variables, utils } from '../../global/helpers';
 import { containerStyles } from '../../global/commonStyles';
 
-const { globalSize } = variables;
+const { marginSize, globalSize } = variables;
+const { media } = utils;
 
 export const HeroHeader = styled.header`
   ${containerStyles};
 
-  padding: ${rem(globalSize.paddingSmall)} ${rem(globalSize.gutter)} 0;
+  padding: ${rem('30px')} ${rem(globalSize.gutter)} ${rem(globalSize.paddingMedium)};
```

Comment:
Tutaj pamietaj ze navigacja i SeoBase daje tez padding zeby to sie zgrywalo, plus wydaje mi sie ze dolny mozna dac 0 skoro illustracja jest na dole, i border-bottom jakis

Reply:
Racja